index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Mobile Portfolio Setup</title>
+</head>
+
+<body>
+    <header class="header padding-left-right">
+        <h2 class="logo">John Doe</h2>
+        <nav class="main-nav">
+            <img class="menu-icon" src="img/menu.svg" alt="Hamburger menu icon" />
+        </nav>
+    </header>
+
+    <section class="headline-section padding-left-right">
+        <div class="headline-box">
+            <div class="headline-text-box">
+                <h1 id="headline-primary">
+                    Hey There. <br /> I'm Alex
+                </h1>
+                <p class="headline-title">I am a software developer</p>
+                <p class="headline-description">
+                    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                <div class="social-icons">
+                    <a href="#">
+                        <img class="icon" src="img/icon-1.svg" alt="github social icon" />
+                    </a>
+
+                    <a href="#">
+                        <img class="icon" src="img/icon-2.svg" alt="linkedin social icon" />
+                    </a>
+
+                    <a href="#">
+                        <img class="icon" src="img/icon-3.svg" alt="pinterest social icon" />
+                    </a>
+
+                    <a href="#">
+                        <img class="icon" src="img/icon-4.svg" alt="twitter social icon" />
+                    </a>
+
+                    <a href="#">
+                        <img class="icon" src="img/icon-5.svg" alt="mercedes social icon" />
+                    </a>
+                </div>
```

Comment:
- [ ] Please wrap all your social links in `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)
- Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.